.github/workflows/publish.yml

```diff
@@ -18,6 +18,8 @@ jobs:
         uses: gradle/gradle-build-action@v3.5.0
       - name: Build
         run: ./gradlew build
+      - name: Generate OpenAPI
+        run: ./gradlew generateOpenAPI
```

Comment:
Are we sure we want to do this?

Given the varied implementations it's not likely to be identical between them. Thus is shouldn't be generated at build time.


Then again maybe we can decide there's a single canonical representation of things for the api.

@agilmore2 thoughts?

Reply:
This provides a good spot to codegen the canonical versions. Generating implementation specific would likely be manually curated anyways, but this would get you 95% of the way there. I would even go so far as to hosting the swagger page as a GitHub Pages site even knowing you couldn't actually test out the queries on a live server, but it still is easier to read than json or yml.